variables.tf

```diff
@@ -58,16 +58,10 @@ variable "existing_sink_name" {
   description = "The name of an existing sink to be re-used for this integration"
 }
 
-variable "existing_pub_sub_topic_id" {
```

Comment:
Why remove these?  Is there no case for a customer to use an existing topic+subscription?

For example, what about a customer that has an existing topic+subscription, but they are not in the Terraform state.

Reply:
This change seems to undo the last PR https://github.com/lacework/terraform-gcp-pub-sub-audit-log/pull/12 which added this as an option

Reply:
I understand you are using this to transition users from v1 to v2 audit log integrations, but this is also the released Terraform for all customers and I am just a bit worried that we have modified this for all those customers who are not migrating too. I know this is controlled by a flag (which is by default `false`), but there is a potential possibly for trouble on ahead with customers setting this erroneously

Reply:
This wasn't in the module before, not even on the CLI side, I only added it last week to help with GCPv2 manual migration. But, we figured out a better way to do that, so I undid it

Reply:
@djmctavish on the CLI side, I am adding this as an additional configuration and explicitly specifying in the prompt that it is for migration only. For onboarding new customers who are not migrating, we can ensure the documentation has clear steps

Reply:
@Manan-Bhatia-0 I think we could see that **what** but not the **why**, it would be useful to attach the migration document as that could give readers that **why** - Good work BTW!